UnitTests/GitCommandsTests/Git/GitCommandHelpersTest.cs

```diff
@@ -605,7 +605,7 @@ public void RebaseCmd()
                 "-c rebase.autoSquash=false rebase -i --no-autosquash \"branch\"",
                 GitCommandHelpers.RebaseCmd("branch", interactive: true, preserveMerges: false, autosquash: false, autoStash: false).Arguments);
             Assert.AreEqual(
-                "-c rebase.autoSquash=false rebase --preserve-merges \"branch\"",
+                "-c rebase.autoSquash=false rebase --rebase-merges \"branch\"",
```

Comment:
Tests should pivot on git version.
So if someone runs a codebase against an older version of git (like me while testing) they shouldn't get a test failure.

Reply:
Note: We have a few tests for features not in the oldest supported version, but those tests do not use Git binaries (only argument tests).